py/specter/util/util.py

```diff
@@ -68,12 +70,18 @@ def __call__(self, x, y):
         # data2 = (self.data[ix-1,iy].T*(1-dx) + self.data[ix,iy].T*dx).T
         # dataxy = (data1.T*(1-dy) + data2.T*dy).T
 
+        #old version
         #- Updated without transposes
         data1 = (self.data[ix-1,iy-1]*(1-dx) + self.data[ix,iy-1]*dx)
         data2 = (self.data[ix-1,iy]*(1-dx) + self.data[ix,iy]*dx)
-        dataxy = (data1*(1-dy) + data2*dy)
+        dataxy_old = (data1*(1-dy) + data2*dy)
+
+        datacopy=self.data        
+
+        dataxy=LinearInterp2D.new_interp(dx,dy,ix,iy,datacopy)
```

Comment:
This looks like work in progress calculating both `dataxy` and `dataxy_old` which should be the same.  Since `new_interp` doesn't involve explicit loops, I'm guessing that numba won't be a big win here, but perhaps a deeper refactor/numbaizing of `__call__` could help.  Let's temporarily restore the `LinearInterp2D` code to the original state, cleanup the rest of this PR and merge it, and then proceed with optimizations of `LinearInterp2D` as a separate study in another branch + PR.

For others reading this: the values of `data1` and `data2` are themselves 2D arrays, so the amount of work being done here is less trivial than it first appears.

Reply:
Yes this was work in progress. Adding JIT to these lines did provide another 4x speedup, but I will work on these changes later. Restored to master for now.